paper/paper.md

```diff
@@ -54,7 +54,40 @@ Representatives of each of these communities have got together to build `FitBenc
 
 `FitBenchmarking` originally started as a tool to benchmark fitting algorithms in the data reduction package `Mantid` [@mantid], which is used to process neutron scattering and muon spectroscopy data. `FitBenchmarking` has since been significantly extended to take data and models from other real world applications and data analysis / modelling / treatment packages, such as `SasView` [@sasview] and `CUTEst` [@cutest]. It fits models to the data by using a range of data fitting and nonlinear optimization software packages, and present comparisons through a variety of different metrics. These include comparison tables and performance profile plots.
 
-`FitBenchmarking` compares how different fitting algorithms perform for the same data, model and initial guess. The best parameters for the model are found by solving a nonlinear least-squares problem, which can either be solved via a fitting package or an off the shelf minimizer. Figure \ref{fig:sample} displays a data set from `FitBenchmarking` where the crosses are the data points and the two curves are the fits found by two optimization algorithms implemented in `GSL` [@gsl]. From Figure \ref{fig:sample}, it is clear that the solution given by lmsder is better. As the volume of data increases, and we do more and more data analysis algorithmically, it is increasingly important that we have the best algorithm without needing to check it by eye. `FitBenchmarking` generates HTML output that makes it easy to compare minimizers on a given problem set.
+As far as we are aware, `FitBenchmarking` is the only package that is designed
+specifically to interface directly with data analysis packages to test different
+implemenations of fitting algorithms.  Below we highlight several packages available that
+are closest to `FitBenchmarking` in spirit.
+
+* `CUTEst` [@cutest] is written in Fortran, but interfaces
```

Comment:
```suggestion
* `CUTEst` [@cutest]  interfaces
```

Reply:
Here would argue it is not relevant/important to mention CUTEst is written in Fortran